tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -182,9 +182,17 @@ func podLogs(labelSelector string, container string) {
 	if err != nil {
 		return
 	}
-	glog.Info("Expect and ignore an error getting crash logs when there are no crash (-p invocation)")
-	_, _ = util.Shell("kubectl --namespace %s logs %s -c %s --tail=40 -p", tc.Kube.Namespace, pod, container)
-	_, _ = util.Shell("kubectl --namespace %s logs %s -c %s --tail=40", tc.Kube.Namespace, pod, container)
+	str, err := util.Shell("kubectl --namespace %s logs %s -c %s --tail=40 -p", tc.Kube.Namespace, pod, container)
```

Comment:
consider using `-l "app=istio-ingress"

Reply:
this is used for all pods, not just ingress

Reply:
I know, but `app=x` is better than `-p` since it collects all pods, not just last terminated.

Reply:
ah interesting, -l is not exactly documented for kubectl logs but
`kubectl logs -l istio=mixer -c mixer` returns something but it may be hard to get which label we get on each, it's also not documented that it gives previous logs without -p (or is it do you have a link for that ?), I tried on mixer but can't tell as there is so much in log before I kill it; it also doesn't seem to die on `kill -SEGV 1` which is odd